lib/mayaUsd/ufe/Utils.h

```diff
@@ -56,6 +56,14 @@ bool isRootChild(const Ufe::Path& path);
 MAYAUSD_CORE_PUBLIC
 SdfLayerHandle defPrimSpecLayer(const UsdPrim& prim);
 
+//! Check if the target layer has any opinions that affects a particular prim
+MAYAUSD_CORE_PUBLIC
+bool isTargetLayerHaveOpinion(const UsdPrim& prim);
```

Comment:
I feel the naming of these two functions is confusing, because they both talk about "target layer", but they're not related.
"isTargetLayerHaveOpinion" is really answering the question: does the edit target layer on the prim's stage have an opinion?
"targetLayerWithOpion" [sic] is answering the question: what is the strongest layer that has an opinion for this prim?  From the point of view of this function, there is no target involved, and therefore I would remove the word "target" here.

Reply:
Please see 54166d2b6

I agree. I made following changes and dropped the target. Let me know if this reads better:

**Before:**
```
isTargetLayerHaveOpinion 
```
**After:**
```
doesLayerHavePrimSpec  
```

"doesLayerHavePrimSpec" answers this question:  Does the layer that you are currently on have any primSpec (a.k.a opinion) on the selected prim?